posixlib/src/main/resources/scala-native/sys/socket.c

```diff
@@ -13,6 +12,26 @@ typedef SSIZE_T ssize_t;
 #else
 #include <netinet/in.h>
 #include <sys/socket.h>
+#if !(defined __STDC_VERSION__) || (__STDC_VERSION__ < 201112L)
+#ifndef SCALANATIVE_SUPPRESS_STRUCT_CHECK_WARNING
+#warning "Size and order of C structures are not checked when -std < c11."
+#endif
+#else
+// Posix defines the order and type of required fields. Size of fields
+// and any internal or tail padding are left unspecified. This section
+// verifies that the C and Scala Native definitons match in each compilation
+// environment.
+//
+// The first sockaddr field in C has had size 2 and no padding after it
+// since time immemorial. Chedk that the Scala Native has the same.
```

Comment:
Wording and spelling of sentence starting with "Chedk".

Reply:
Both fixed, along with another typo.  Thank you.